### DISCUSSION TITLE
Theme list not displaying with :theme command

### DISCUSSION THREAD

**Discussion by Miguel-A-Jara** [helix-editor/helix#10810] [Troubleshooting] [answered]
I have a fresh install using fedora's package manager `dnf`.
In order to reproduce this, I have made this set of steps:

1. Run: `sudo dnf install helix`
2. Open helix with the `hx` command.
3. Run the `:themes` command.
After doing that, only the default theme shows up.

Now, running `:tutor` works fine with no problems, so I'm guessing my runtime directory is set up correctly.
After running `find / -iname tutor` on linux this is the relevant path found:

`/usr/lib64/helix/runtime/tutor`.
An `ls` of the `runtime` directory outputs this:

`grammars queries themes tutor`

And themes does have the different `.toml` files that come by default, e.g: `acme.toml, adwaita-dark.toml`, etc.
![image](https://github.com/helix-editor/helix/assets/95490111/ed35a186-fd69-48a0-9d35-472882d37fca)

Could someone help me with this :)? I've been searching on the issues & discussion but couldn't find anything that solved my problem

**Comment by krish-r** [comment, marked as answer]
Can you try `:theme ` (`:theme` followed by a `<SPACE>`), it should display the themes. Without an argument it will just display your current theme.

![hx_theme](https://github.com/helix-editor/helix/assets/54745129/232279e9-9bad-4ef9-bf91-dad6899a86df)

(The runtime directory looks correct to me.)

**Reply by Miguel-A-Jara** [reply]
That's so silly of me... It works as expected, thank you!

**Reply by OscarRaizer** [reply]
> That's so silly of me... It works as expected, thank you!

hahah I reinstalled everything about 5 times until I found this

**Reply by ulvido** [reply]
same here T-T
